code/modules/games/spaceball_cards.dm

```diff
@@ -8,7 +8,7 @@
 	..()
 	var/datum/playingcard/P
 	var/i
-	var/year = 554 + text2num(time2text(world.timeofday, "YYYY"))
+	var/year = 310 + text2num(time2text(world.timeofday, "YYYY"))
```

Comment:
I get the feeling this was originally an oversight and you probably shouldn't keep it 10 years in the future of the present. They probably don't make playing cards for sports teams 10 years in the future of the current date!